core/include/tangram/tangram.h

```diff
@@ -63,6 +63,21 @@ struct SceneUpdate {
     SceneUpdate(std::string p, std::string v) : path(p), value(v) {}
 };
 
+enum SceneUpdateError {
```

Comment:
Yes sounds good, I can change that for a `TangramError` which where we could add more errors incoming from tangram API use.